cypress/support/commands.ts

```diff
@@ -62,15 +62,17 @@ Cypress.Commands.add(
 );
 
 Cypress.Commands.add("login", () => {
-  cy.wait(4000); //prevent random failure
-  cy.get("#rh-password-verification-submit-button").then(($item) => {
-    if ($item.is(":visible")) {
-      cy.log("Cookie's choise is not present");
-    } else {
-      cy.log("Cookie's choise is present");
-      cy.get("button:contains('Accept default')").click().should("not.exist");
-    }
-  });
+  //https://docs.cypress.io/guides/core-concepts/conditional-testing#Element-existence
+  cy.get("body", { timeout: 10000 })
+    .should("be.visible")
+    .then(($body) => {
+      if ($body.find('div[class="truste_overlay"]').length) {
+        // the form was found, do something else here
+        cy.get("button:contains('Accept default')").click().should("not.exist");
+      } else {
+        cy.log("Cookie's choise is not present");
+      }
+    });
```

Comment:
I am approving this snippet without further checking. I struggle to get cookies dialogue locally so it is just my trust that locators are correct. However as @sjamboro wrote, it doesn't seems to be like 'permanent' solution